QEfficient/exporter/export_utils.py

```diff
@@ -17,6 +17,24 @@
 import torch
 from onnx import external_data_helper, numpy_helper
 
+# import onnxsim
+# def simplify_onnx(gen_models_path: str, model_base_name: str, **kwargs) -> str:
+#     try:
+#         simple_model, check = onnxsim.simplify(
+#             f"{gen_models_path}/{model_base_name}.onnx", **kwargs
+#         )
+#         assert check, "Failed verification of simplified model"
+#         return save_onnx(simple_model, gen_models_path, model_base_name + "_simplified")
+#     except Exception as e:
+#         print(f"Skipping simplifier: {e}")
+#         onnx.shape_inference.infer_shapes_path(
+#             f"{gen_models_path}/{model_base_name}.onnx",
+#             f"{gen_models_path}/{model_base_name}_inf.onnx",
+#             True,
+#             True,
+#             True,
+#         )
+#         return save_onnx(model_base_name + "_inf", gen_models_path, model_base_name)
```

Comment:
We can just remove this and create a function for shape_inference only, as onnxsim creates more issues that it solves. Can be done in a future PR too.